data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/configuration/S3ScanScanOptions.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.source.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+/**
+ * Class consists the scan options list bucket configuration properties.
+ */
+public class S3ScanScanOptions {
+    @JsonProperty("range")
+    private String range;
```

Comment:
Based on previous comments, this should be a `Duration`.

Data Prepper can parse common strings into `Duration` using the `DataPrepperDurationDeserializer`.